packages/thegraph-data-access/src/data-access.ts

```diff
@@ -8,20 +8,25 @@ import { Block } from '@requestnetwork/data-access';
 import { DataAccessTypes, LogTypes, StorageTypes } from '@requestnetwork/types';
 
 import { Transaction } from './queries';
-import { SubgraphClient } from './subgraphClient';
-import { TheGraphStorage } from './TheGraphStorage';
-import { CombinedDataAccess } from './CombinedDataAccess';
-import { PendingStore } from './PendingStore';
+import { SubgraphClient } from './subgraph-client';
+import { CombinedDataAccess } from '@requestnetwork/data-access';
+import { PendingStore } from './pending-store';
 
-export type TheGraphDataAccessOptions = {
-  ipfsStorage: StorageTypes.IIpfsStorage;
-  graphql: { url: string } & RequestInit;
-  signer: Signer;
+type TheGraphDataAccessBaseOptions = {
   network: string;
   logger?: LogTypes.ILogger;
   pendingStore?: PendingStore;
 };
 
+export type TheGraphDataAccessOptions = TheGraphDataAccessBaseOptions & {
```

Comment:
```suggestion
export type TheGraphDataReadOptions = TheGraphDataAccessBaseOptions & {
```
Might make the difference with the other type more explicit?

Reply:
It's also consistent with the  `TheGraphDataRead` class name

Reply:
i removed datawrite and moved the `storage` to `TheGraphDataAccessOptions`, which now makes sense like this.